js/app.js

```diff
@@ -95,3 +95,22 @@ createTable();
 for (var i = 0; i < allStores.length; i++) {
   allStores[i].generateTableRow();
 }
+
+var elStoreForm = document.getElementById('newStoreForm');
+
+function submitButton(event) {
+  event.preventDefault();
+  var newStoreForm = event.target;
```

Comment:
You don't need to declare yet another variable to point at the `form`. That's what you already did on line 99